org.eclipse.winery.topologymodeler.ui/src/app/node/node.component.ts

```diff
@@ -143,44 +142,44 @@ export class NodeComponent implements OnInit, AfterViewInit, OnDestroy, DoCheck
      * We have 3 possibilities: none, XML element, or Key value pairs.
      * @param {string} type
      */
-    findOutPropertyDefinitionTypeForProperties(type: string, groupedNodeTypes: any): string {
-        for (const nameSpace of groupedNodeTypes) {
-            for (const nodeTypeVar of nameSpace.children) {
+    findOutPropertyDefinitionTypeForProperties(type: string, groupedNodeTypes: any): void {
```

Comment:
Why is `groupedNodeTypes` of type `any`?

Reply:
Oh I already saw these changes in https://github.com/eclipse/winery/pull/249. What did you do here?

Reply:
done